gearmand/tests/test_integration.py

```diff
@@ -0,0 +1,36 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.gearmand import Gearman
+
+from . import common
+
+
+@pytest.mark.usefixtures("dd_environment")
+def test_service_check(check, aggregator):
```

Comment:
Should you remove that test? It's almost exactly the same as the other one, except it calls check once.

Reply:
Yes, right, it's duplicate, no much value having it.